devdoc/constbuffer_array_requirements.md

```diff
@@ -54,6 +55,24 @@ MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_create, const CO
 
 **SRS_CONSTBUFFER_ARRAY_01_014: [** If any error occurs, `constbuffer_array_create` shall fail and return NULL. **]**
 
+### constbuffer_array_create_with_move_memory
+
+```c
+MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_create_with_move_memory, CONSTBUFFER_HANDLE*, buffers, uint32_t, buffer_count);
```

Comment:
>constbuffer_array_create_with_move [](start = 46, length = 34)

So there's this... question... about WHAT is moved... (I just realized). Following code:

CODE1:
```c 
CONSTBUFFER_HANDLE h1= _create();
CONSTBUFFER_HANDLE h2= _create();
CONSTBUFFER_HANDLE* h12 = malloc(2*sizeof(CONSTBUFFER_HANDLE));

constbuffer_array_create_with_move_memory(h12, 2). 
```
Who does the free(h12)?
Who does the CONSTBUFFER_DecRef(h1), CONSTBUFFER_DecRef(h2)?


CODE2:
```c
CONSTBUFFER_HANDLE h12[2];
h12[0] = _create();
h12[1] = _create();

constbuffer_array_create_with_move_memory(h12, 2)
```
Is Code2 valid? Who does CONSTBUFFER_DecRef(h12[0])?